### PR TITLE
Übersetze how-to-contribute

### PR DESCRIPTION
**Pull request by katrinleinweber**

### CODE REVIEW

**Comment by jens1o** [comment]
ping me if you want a review

**Comment by katrinleinweber** [comment]
You're always welcome to review :-)

File: _articles/de/how-to-contribute.md [hunk 1]
```diff
@@ -1,220 +1,242 @@
 ---
 lang: de
 title: Wie zu Open Source beitragen?
-description: Want to contribute to open source? A guide to making open source contributions, for first-timers and for veterans.
+description: Möchten Sie zu Open Source beitragen? Hier ein Leitfaden dafür für Einsteiger und Veteraninnen.
```
Comment (jens1o) [inline _articles/de/how-to-contribute.md]: `Hier finden Sie ein Leitfaden für Einsteiger und Fortgeschrittene.`?

File: _articles/de/how-to-contribute.md [hunk 2]
```diff
@@ -1,220 +1,242 @@
...
+### Bestehende Fähigkeiten verbessern
 
-Whether it's coding, user interface design, graphic design, writing, or organizing, if you're looking for practice, there's a task for you on an open source project.
+Ob Programmierung, User Interface Design, Grafikdesign, Schreiben oder Organisieren: Wenn Sie auf der Suche nach Praxiserfahrung sind, werden Sie Aufgaben für sich in einem Open-Source-Projekt finden.
 
-### Meet people who are interested in similar things
+### Treffen Sie Menschen, die an ähnlichen Dingen interessiert sind
 
-Open source projects with warm, welcoming communities keep people coming back for years. Many people form lifelong friendships through their participation in open source, whether it's running into each other at conferences or late night online chats about burritos.
+Open-Source-Projekte mit warmherzigen, einladenden Communities schaffen langfristige Hobbies für viele Leute. Lebenslange Freundschaften können durch ihre Teilnahme an Open Source entstehen, sei es bei Konferenzen oder bei nächtlichen Online-Chats über Lamacuns.
```
Comment (jens1o) [inline _articles/de/how-to-contribute.md]: What is `Lamacuns`? :D

File: _articles/de/how-to-contribute.md [hunk 3]
```diff
@@ -1,220 +1,242 @@
...
 
-### Find mentors and teach others
+### Mentoren finden und unterrichte Andere
 
-Working with others on a shared project means you'll have to explain how you do things, as well as ask other people for help. The acts of learning and teaching can be a fulfilling activity for everyone involved.
+Wenn Sie mit Anderen an einem gemeinsamen Projekt arbeiten, müssen Sie erklären, wie Sie dabei vorgehen und gleichzeitig andere Menschen um Hilfe bitten. Das Lernen und Lehren kann eine erfüllende Tätigkeit für alle Beteiligten sein.
 
-### Build public artifacts that help you grow a reputation (and a career)
+### Öffentliche Ergebnisse zeugen von Ihrer Arbeit und fördern Ihren Ruf (und Ihre Karriere)
```
Comment (jens1o) [inline _articles/de/how-to-contribute.md]: `Ihrer`

File: _articles/de/how-to-contribute.md [hunk 4]
```diff
@@ -1,220 +1,242 @@
...
+Dies ist ein weit verbreiteter Irrtum, aber in Wirklichkeit  sind es oft andere Projektaspekte, die am [meisten Unterstützung gebrauchen können](https://github.com/blog/2195-the-shape-of-open-source). Sie tun dem Projekt einen _großen_ Gefallen, indem Sie anbieten, bei nicht-Code-Aspekten mitzuwirken!
 
 <aside markdown="1" class="pquote">
   <img src="https://avatars.githubusercontent.com/orta?s=180" class="pquote-avatar" alt="avatar">
-  I’ve been renowned for my work on CocoaPods, but most people don’t know that I actually don’t do any real work on the CocoaPods tool itself. My time on the project is mostly spent doing things like documentation and working on branding.
+  
+  Ich bin bekannt für meine Arbeit an CocoaPods, aber die meisten Leute wissen nicht, dass ich eigentlich keine echte Arbeit am CocoaPods-Tool selbst mache. Meine Zeit im Projekt verbringe ich hauptsächlich mit Dokumentation und Markendesign.
```
Comment (jens1o) [inline _articles/de/how-to-contribute.md]: `Marketing`?

File: _articles/de/how-to-contribute.md [hunk 5]
```diff
@@ -1,220 +1,242 @@
...
 
-* Link to duplicate issues, and suggest new issue labels, to keep things organized
-* Go through open issues and suggest closing old ones, [like @nzakas did for ESLint](https://github.com/eslint/eslint/issues/6765)
-* Ask clarifying questions on recently opened issues to move the discussion forward
+* Verlinken Sie doppelte Issues und schlagen Sie neue Labels vor, um den Issue Tracker in Ordnung zu halten
+* Gehen Sie offene Fragen durch und schlagen Sie alte zur Schließung vor, wie @nzakas [in ESLint](https://github.com/eslint/eslint/issues/6765)
+* Stellen Sie konstruktive Fragen zu kürzlich eröffneten Issues, um Diskussionen voranzubringen
 
-### Do you like to code?
+### Mögen Sie es zu kodieren?
```
Comment (jens1o) [inline _articles/de/how-to-contribute.md]: Machine translation?

`Mögen Sie es zu programmieren?`

File: _articles/de/how-to-contribute.md [hunk 6]
```diff
@@ -1,220 +1,242 @@
...
 
-* Find an open issue to tackle, [like @dianjin did for Leaflet](https://github.com/Leaflet/Leaflet/issues/4528#issuecomment-216520560)
-* Ask if you can help write a new feature
-* Automate project setup
-* Improve tooling and testing
+* Finden Sie ein offenes Issues, das Sie bearbeiten können, wie @dianjin [in Leaflet](https://github.com/Leaflet/Leaflet/issues/4528#issuecomment-216520560)
```
Comment (jens1o) [inline _articles/de/how-to-contribute.md]: `Issue`

File: _articles/de/how-to-contribute.md [hunk 7]
```diff
@@ -1,220 +1,242 @@
...
 
-* Answer questions about the project on e.g., Stack Overflow ([like this Postgres example](https://stackoverflow.com/questions/18664074/getting-error-peer-authentication-failed-for-user-postgres-when-trying-to-ge)) or Reddit
-* Answer questions for people on open issues
-* Help moderate the discussion boards or conversation channels
+* Beantworten Sie Fragen zum Projekt, z.B. auf Stack Overflow ([wie dieses Postgres-Beispiel zeigt](https://stackoverflow.com/questions/18664074/getting-error-peer-authentication-failed-for-user-postgres-when-trying-to-ge)) oder auf Reddit
+* Beantworten Sie Fragen in offenen Issues
+* Helfen Sie bei der Moderation von Diskussionsforen oder anderen Kommunikationskanälen
 
-### Do you like helping others code?
+### Helfen Sie gerne anderen beim Programmieren?
```
Comment (jens1o) [inline _articles/de/how-to-contribute.md]: `Anderen`

File: _articles/de/how-to-contribute.md [hunk 8]
```diff
@@ -1,220 +1,242 @@
...
+Während sich "Open Source" oft auf Software bezieht, kann man an fast allen anderen Arten von Projekten zusammenarbeiten: Bücher, Rezepte, Listen, Kurse, uvm. werden als Open-Source-Projekte entwickelt.
 
-For example:
+Zum Beispiel:
 
-* @sindresorhus curates a [list of "awesome" lists](https://github.com/sindresorhus/awesome)
-* @h5bp maintains a [list of potential interview questions](https://github.com/h5bp/Front-end-Developer-Interview-Questions) for front-end developer candidates
-* @stuartlynn and @nicole-a-tesla made a [collection of fun facts about puffins](https://github.com/stuartlynn/puffin_facts)
+* @sindresorhus kuratiert eine [Liste von "awesome" -Listen](https://github.com/sindresorhus/awesome)
+* @h5bp unterhält eine [Liste möglicher Bewerbungstests](https://github.com/h5bp/Front-end-Developer-Interview-Questions) für Frontend-Entwickler-Kandidat\*innen
```
Comment (jens1o) [inline _articles/de/how-to-contribute.md]: `Bewerbungsfragen`

File: _articles/de/how-to-contribute.md [hunk 9]
```diff
@@ -225,304 +247,388 @@ You can also use one of the following resources to help you discover and contrib
 * [Up For Grabs](http://up-for-grabs.net/)
 * [Contributor-ninja](https://contributor.ninja)
```
Comment (jens1o) [inline _articles/de/how-to-contribute.md]: I'd add `(Englisch)` to all of them

File: _articles/de/how-to-contribute.md [hunk 10]
```diff
@@ -225,304 +247,388 @@ You can also use one of the following resources to help you discover and contrib
...
+    
   </label>
 </div>
 
 <div class="clearfix mb-4">
   <input type="checkbox" id="cbox9" class="d-block float-left mt-1 mr-2" value="checkbox">
   <label for="cbox9" class="overflow-hidden d-block text-normal">
-    Are issues getting closed? (On GitHub, click the "closed" tab on the Issues page to see closed issues.)
+    
+    Werden Issues  auch wieder geschlossen? (Klicken Sie bei GitHub auf der Seite "Issues" auf "closed", um geschlossene Issues zu sehen.)
```
Comment (jens1o) [inline _articles/de/how-to-contribute.md]: unnecessary space

File: _articles/de/how-to-contribute.md [hunk 11]
```diff
@@ -225,304 +247,388 @@ You can also use one of the following resources to help you discover and contrib
...
+> 😇 _"Danke, dass Sie sich diesen Fehler angesehen haben. Ich bin Ihren Ratschlägen gefolgt. Hier ist die Ausgabe."_
 >
-> 😢 _"Why can't you fix my problem? Isn't this your project?"_
+> 😢 _"Warum kannst Du mein Problem nicht lösen? Ist das nicht Dein Projekt?"_
 
-**Respect community decisions.** Your ideas may differ from the community's priorities or vision. They may offer feedback or decide not to pursue your idea. While you should discuss and look for compromise, maintainers have to live with your decision longer than you will. If you disagree with their direction, you can always work on your own fork or start your own project.
+**Respektieren Sie die Entscheidungen der Gemeinschaft ** Ihre Ideen könnten von den Prioritäten oder der Vision der Gemeinschaft abweichen. Diese gibt Ihnen vielleicht eine Rückmeldung oder beschließt, Ihre Idee nicht weiterzuverfolgen. Auch wenn Sie Kompromisse suchen und besprechen sollten, müssen die Maintainer\*innen mit einer Entscheidung länger leben als Sie selbst. Wenn Sie mit deren Richtung nicht einverstanden sind, können Sie jederzeit an Ihrem eigenen Fork arbeiten oder Ihr eigenes Projekt starten.
```
Comment (jens1o) [inline _articles/de/how-to-contribute.md]: Unnecessary space breaking formatting.

File: _articles/de/how-to-contribute.md (hunk 2 above)
Comment (katrinleinweber) [reply _articles/de/how-to-contribute.md]: A Turkish dish that I'd consider as somewhat equivalent to what burritos are for Americans.

File: _articles/de/how-to-contribute.md (hunk 3 above)
Comment (katrinleinweber) [reply _articles/de/how-to-contribute.md]: Mir scheint sich das auf `fördern` zu beziehen.

File: _articles/de/how-to-contribute.md (hunk 2 above)
Comment (jens1o) [reply _articles/de/how-to-contribute.md]: looks like it is called `Lahmacun` in German.

File: _articles/de/how-to-contribute.md [hunk 12]
```diff
@@ -41,7 +41,7 @@ Ob Programmierung, User Interface Design, Grafikdesign, Schreiben oder Organisie
 
 ### Treffen Sie Menschen, die an ähnlichen Dingen interessiert sind
 
-Open-Source-Projekte mit warmherzigen, einladenden Communities schaffen langfristige Hobbies für viele Leute. Lebenslange Freundschaften können durch ihre Teilnahme an Open Source entstehen, sei es bei Konferenzen oder bei nächtlichen Online-Chats über Lamacuns.
+Open-Source-Projekte mit warmherzigen, einladenden Communities schaffen langfristige Hobbys für viele Leute. Lebenslange Freundschaften können durch ihre Teilnahme an Open Source entstehen, sei es bei Konferenzen oder bei nächtlichen Online-Chats über Lhamacuns.
```
Comment (jens1o) [inline _articles/de/how-to-contribute.md]: `Lahmacuns`

Comment (katrinleinweber) [reply _articles/de/how-to-contribute.md]: m-)

Comment (jens1o) [reply _articles/de/how-to-contribute.md]: Enjoy your time off!

Comment (katrinleinweber) [reply _articles/de/how-to-contribute.md]: I shall, but I'll not be 100% offline. Please merge when you think this is ready, or let me know when I shall merge.

**Pull request merged**